db/migrate/20200522105202_add_comment_to_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddCommentToPackages < ActiveRecord::Migration
+  def change
+    add_column :packages, :comment, :text
```

Comment:
We actually already have a 'notes' field. Talking with Matt, we don't need to add a separate field. Just use 'notes' and don't create a new comments field.